### DISCUSSION TITLE
umi3 引入一个npm包，这个npm包引入了一个scss 在umi3支持scss的情况下，scss样式丢了

### DISCUSSION THREAD

**Discussion by zhousufa** [umijs/umi#7966] [Q&A]
自己发布了一个npm包，npm包通过css module的方式引入了一个scss `import styles from 'index.module.scss'` 在npm包的开发过程中这个样式是可以的，本地demo也是好的。
发布成功后在es目录里有index.js 和index.module.scss

在umi3的工程里引入这个npm包，发现样式不生效，debug进去发现import styles from 'index.module.scss'返回的是空对象

**Comment by jiyuyu** [comment]
请问解决了吗
